bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -34,64 +36,64 @@ def __init__(self, component, test_config, bundle_manifest, work_dir, s3_bucket_
         )
 
     def execute(self):
-        self._fetch_plugin_specific_dependencies()
+        self.__install_build_dependencies()
         for config in self.test_config.integ_test["test-configs"]:
-            self._setup_cluster_and_execute_test_config(config)
+            security = self.__is_security_enabled(config)
+            self.__setup_cluster_and_execute_test_config(security)
 
-    # TODO: fetch pre-built dependencies from s3
-    def _fetch_plugin_specific_dependencies(self):
-        dependencies_dir = os.path.join(self.work_dir, 'dependencies')
-        os.chdir(self.work_dir)
-        subprocess.check_output(
-            "cp -r dependencies/job-scheduler " + self.component.name, cwd=self.work_dir, shell=True
-        )
-        job_scheduler_dir = os.path.join(self.work_dir, self.component.name, 'job-scheduler')
-        os.chdir(job_scheduler_dir)
-        deps_script = os.path.join(
-            self.work_dir,
-            "opensearch-build/tools/standard-test/integtest_dependencies_opensearch.sh",
-        )
-        subprocess.check_output(
-            f"{deps_script} job-scheduler {self.bundle_manifest.build.version}",
-            cwd=job_scheduler_dir,
-            shell=True
-        )
-        os.chdir(dependencies_dir)
-        if not (os.path.isdir('notifications')):
-            subprocess.check_output("cp -r alerting notifications", cwd=dependencies_dir, shell=True)
-        notifications_dir = os.path.join(dependencies_dir, 'notifications')
-        os.chdir(notifications_dir)
-        subprocess.check_output(
-            f"{deps_script} alerting {self.bundle_manifest.build.version}",
-            cwd=notifications_dir,
-            shell=True
-        )
+    def __install_build_dependencies(self):
+        if 'build-dependencies' in self.test_config.integ_test:
+            dependency_list = self.test_config.integ_test['build-dependencies']
+            if len(dependency_list) == 1 and 'job-scheduler' in dependency_list:
+                self.__copy_job_scheduler_artifact()
+            else:
+                raise InvalidTestConfigError(
+                    'Integration test job only supports job-scheduler build dependency at present.')
```

Comment:
Didn't get this part. Are we not supporting other dependencies like alerting, common-utils, etc?

Reply:
those are maven dependencies, still supported. This is for build dependencies, like job-scheduler zip